mmdet/apis/inference.py

```diff
@@ -29,6 +29,8 @@ def init_detector(config, checkpoint=None, device='cuda:0'):
     elif not isinstance(config, mmcv.Config):
         raise TypeError('config must be a filename or Config object, '
                         f'but got {type(config)}')
+    if update_config:
+        config.merge_from_dict(update_config)
```

Comment:
Is it possible to load config and update it at a script level, not inside the API code?

Reply:
done